frameworks/g-cloud-9/questions/services/otherNetworks.yml

```diff
@@ -0,0 +1,16 @@
+name: "Other networks"
+question: "Can your service be connected to any other networks?"
+question_advice: "Say 'yes' if you can connect to a private WAN."
```

Comment:
Is that a "for example", or would it just be easier to ask "can you connect to a private WAN" as the question?